### PR TITLE
Display database error when it occurs during installation

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
Currently on the installation script, the database step does not check if an error is displayed and tries to continue anyway.

We must display it and stop the script execution in that case. This PR shows what happens.

Current example for https://github.com/PrestaShop/PrestaShop/pull/8412
capture du 2017-11-17 10-45-50

### CODE REVIEW

File: test/itg/1.7/install_prestashop.js [hunk 1]
```diff
@@ -137,6 +137,11 @@ function runScenario() {
                     .click(this.selector.InstallationWizardPage.test_conection_button)
                     .waitForExist(this.selector.InstallationWizardPage.dbResultCheck_green_block, 300000)
                     .call(done);
+                this.client
+                    .waitForExist(this.selector.InstallationWizardPage.dbResultCheck_red_block, 3000)
+                    .getText(this.selector.InstallationWizardPage.dbResultCheck_red_block).then(function (text) {
+                        console.error("Could not confirm database credentials.", text);
+                    });
```
Comment (Quetzacoalt91) [inline test/itg/1.7/install_prestashop.js:136]: @fatmaBouchekoua do you know how to stop the test? We can't go further if the database is not validated, and PrestaShop is not installed.

Comment (fatmaBouchekoua) [reply test/itg/1.7/install_prestashop.js:136]: Try ```this.client.end()``` to end the session and close the browser

Comment (Quetzacoalt91) [reply test/itg/1.7/install_prestashop.js:136]: Unfortunately the script was still running :(

**Comment by Quetzacoalt91** [comment]
I'm done with this PR. I let your review it @fatmaBouchekoua before integrating it on your branch.

**Pull request closed without merging**